contracts/finance/andromeda-validator-staking/src/contract.rs

```diff
@@ -451,9 +450,11 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractEr
     }
 }
 
+#[allow(deprecated)]
 pub fn on_validator_unstake(deps: DepsMut, msg: Reply) -> Result<Response, ContractError> {
     let res = msg.result.unwrap();
     let mut unstaking_queue = UNSTAKING_QUEUE.load(deps.storage).unwrap_or_default();
+    // TODO this is deprecated
```

Comment:
Use `res.msg_responses` if `res.data` does not exist (chain is running CW2.0)

Reply:
How do we utilize msg_response's data? Tried to use `decode_unstaking_response_data` like for `res.data` but it didn't work.